static/js/bootstrapper.js

```diff
@@ -83,6 +81,22 @@ var utils = require('kujua-utils');
     return callback(err);
   };
 
+  var hasRole = function(userCtx, role) {
```

Comment:
Do you want to note where you've duplicated this code in the code itself, so we don't forget what to pull out once we have a solution to this?

Reply:
Added comments everywhere this is used.